tests/test_kasiopea.py

```diff
@@ -217,17 +217,6 @@ def modify_task(self):
         overwrite_file(self.task_dir, "check.py", "check_strict.py")
 
 
-class TestBigInput(TestSumKasiopea):
-    def expecting_success(self):
-        return False
-
-    def modify_task(self):
-        def modification_fn(raw_config):
-            raw_config["limits"]["input_max_size"] = "0"
-
-        modify_config(self.task_dir, modification_fn)
-
-
```

Comment:
Is there a reason we can't keep these configs?

Reply:
I don't know about these. Cause now zero means unlimited testing this would probably mean stuffing the inputs with whitespace, which I don't particularly like. I am thinking more about adding new fixtures and keeping these for testing v2->v3 conversion. What do you think?

Reply:
That's fair, all of our fixtures have inputs and outputs less than a megabyte in size.
I guess we could modify `odd_stub` to generate bigger inputs pretty easily, through.

Reply:
Done